tests/unit/lms/services/jwt_test.py

```diff
@@ -165,12 +163,6 @@ def encode_jwt(
 
         return jwt.encode(payload, secret, algorithm=algorithm, headers=headers)
 
-    @pytest.fixture(autouse=True)
```

Comment:
Found by @mtomilov here https://hypothes-is.slack.com/archives/C1MA4E9B9/p1746022871655979